be/src/olap/memtable_flush_executor.cpp

```diff
@@ -42,7 +44,7 @@ OLAPStatus FlushHandler::wait() {
     return _last_flush_status.load();
 }
 
-void FlushHandler::on_flush_finished(const FlushResult& res) {
+void FlushHandler::_on_flush_finished(const FlushResult& res) {
```

Comment:
Those `on_xxx` methods of `FlushHandler` should be public. FlushHandler is independent of MemTableFlushExecutor. Just because FlushHandler is a friend class of MemTableFlushExecutor, so that you can change it to private.

Reply:
It is precisely because these `on_xxx()` methods can only be called by `MemTableFlushExecutor`, so we should set them to be `private`, that is, they should not be exposed to users.

Reply:
Anyway, because this code will be refactored soon, and future `FlushHandler` (may be `FlushToken`) will only expose the `submit()` and `wait()` interfaces, so now I won't change it here.